internal/backends/qt/qt_window.rs

```diff
@@ -219,14 +219,10 @@ cpp! {{
         }
 
         void closeEvent(QCloseEvent *event) override {
-            bool accepted = rust!(Slint_requestClose [rust_window: &QtWindow as "void*"] -> bool as "bool" {
-                return WindowInner::from_pub(&rust_window.window).request_close();
+            rust!(Slint_requestClose [rust_window: &QtWindow as "void*"] {
+                rust_window.window.dispatch_event(WindowEvent::CloseRequested);
             });
-            if (accepted) {
-                event->accept();
-            } else {
-                event->ignore();
-            }
+            event->ignore();
```

Comment:
I've found it interesting that the entire accept/ignore business of QCloseEvent really exists only to support `bool QWidget::close()`. It seems the platform plugin entirely ignores the result (which makes sense).